var/spack/repos/builtin/packages/samtools/package.py

```diff
@@ -37,6 +38,11 @@ class Samtools(Package):
     version('1.3.1', 'a7471aa5a1eb7fc9cc4c6491d73c2d88')
     version('1.2', '988ec4c3058a6ceda36503eebecd4122')
 
+    # For older packages that depend on samtools, this variant includes
```

Comment:
Is it safe for older packages that are expecting the old structure to use the new samtools?  Would it be safer to make packages that expect the old structure depend on the older samtools?

If it *is* safe, perhaps it's useful to make this Variant default to True, or just do away with the variant and install the extra bits just in case?